.decent_ci-Linux.yaml

```diff
@@ -5,7 +5,7 @@ compilers:
     release_build_enable_pgo: true
     release_build_training_cmake_extra_flags: -DPROFILE_GENERATE:BOOL=ON -DENABLE_REGRESSION_TESTING:BOOL=OFF
     release_build_training_ctest_filter:  -R  "integration.(RefBldg.*|.*Slab|.*[^g]Basement|.*5Zone.*|Rad|Pip|.*Network.*|Window|CentralChillerHeaterSystem_Cooling_Heating|CmplxGlz_.*|ConvectionAdaptiveSmallOffice|DElight.*|DaylightingDevice.*|DisplacementVent_Nat_AirflowNetwork_AdaptiveComfort|DOADualDuctSchool|EMS.*|Flr_Rf_8Sides|FluidCooler|GSHP.*|GeneratorswithPV|HeatPumpWaterToAir.*|HospitalBaselineReheatReportEMS|HospitalLowEnergy|MicroCogeneration|PlantApplicationsGuide.*|PlateHeatExchanger|PurchAirWithDaylighting|RefrigeratedWarehouse|SolarCollectorFlatPlateWater|SurfaceTest|ThermalChimneyTest|UserDefinedRoomAirPatterns|VSHeatPumpWaterToAirEquationFit|WaterHeater.*|_CTFTestsPart2|_HybridVentilationControlGlobalAN|CentralChillerHeaterSystem)"
-    release_build_cmake_extra_flags: -DPROFILE_GENERATE:BOOL=OFF -DPROFILE_USE:BOOL=ON -DENABLE_LTO:BOOL=ON -DENABLE_REGRESSION_TESTING:BOOL=ON
```

Comment:
Oh, and is everyone (@kbenne, @lefticus) OK with this removal of LTO?  Given the short time before release, I think it's the safest option then we can re-investigate it after release and try to make the source/build changes to get it working...